index.html

```diff
@@ -97,18 +138,25 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
     $searchDiv.search('query').search('show results')
   ;
   })
+  function indexCollection(item, name) {
+    item[name] = item[name].join('').replace(" ", "")
```

Comment:
I did a few test with them and if your query has two words, then every thing match !